critiquebrainz/frontend/external/musicbrainz.py

```diff
@@ -196,3 +217,14 @@ def get_entity_by_id(id, type='release_group'):
     elif type == 'place':
         rv = get_place_by_id(id)
     return rv
+
+
+def get_url_rels_from_releases(releases):
+    """Returns all url-rels for a list of releases in a single list (of url-rel dictionaries)
+    Typical usage with browse_releases()
+    """
+    all_url_rels = []
+    for release in releases['release-list']:
```

Comment:
@gentlecat The error I was referring to you is raised here.

```
File "/code/critiquebrainz/frontend/external/musicbrainz.py", line 227, in get_url_rels_from_releases
Open an interactive python shell in this frame 
for release in releases['release-list']:
```

I checked by adding a few print statements in the code, apparently it returns a list only

https://dpaste.de/LAkx